### PR TITLE
Fix issues cause CI fails

### PR DESCRIPTION
**Pull request by pschang-phy**
### Ticket
N/A

### Problem description
CI fails on

- opencv dependency https://github.com/tenstorrent/pytorch2.0_ttnn/actions/runs/11966694211/job/33411174595

- ttnn.sum issues https://github.com/tenstorrent/pytorch2.0_ttnn/actions/runs/11982764756/job/33411292571#step:4:4486
Found the `ttnn.sum` ran into wrong results (also happened in local machine). But after reset board n-150, it backs to normal.

### What's changed
- Try fixing dependency error of yolov5
- Skip `ttnn.sum` related tests temporarily

### CODE REVIEW

File: requirements-dev.txt [hunk 1]
```diff
@@ -17,3 +17,4 @@ diffusers
 pytorchcv
 librosa
 soundfile
+opencv-python-headless
```
Comment (jerrysky3) [inline requirements-dev.txt:20]: Maybe we can pin the package version? So the package won't change by surprise

File: tests/lowering/misc/test_if.py [hunk 2]
```diff
@@ -19,6 +19,7 @@ def forward(self, x):
     "input_shape",
     [(4, 4)],
 )
+@pytest.mark.skip(reason="Temporarily disabled due to reset issue in ttnn.sum")
```
Comment (jerrysky3) [inline tests/lowering/misc/test_if.py:22]: If there is an issue to track this issue, the issue number can be added here so we won't turn them back after the problem is resolved

Comment (ayerofieiev-tt) [reply tests/lowering/misc/test_if.py:22]: agreed

File: requirements-dev.txt (hunk 1 above)
Comment (ayerofieiev-tt) [reply requirements-dev.txt:20]: I am ok to allow a similar problem to appear again before we decide to pin the package version.
Wdyt?

File: tests/lowering/misc/test_if.py (hunk 2 above)
Comment (pschang-phy) [reply tests/lowering/misc/test_if.py:22]: I fire this issue https://github.com/tenstorrent/pytorch2.0_ttnn/issues/499 to track

File: tests/lowering/misc/test_sum.py [hunk 3]
```diff
@@ -23,6 +23,7 @@ def forward(self, tensor):
         (1,),
     ],
 )
+@pytest.mark.skip(reason="Temporarily disabled due to issue#499")
```
Comment (jerrysky3) [inline tests/lowering/misc/test_sum.py:26]:
```suggestion
@pytest.mark.skip(reason="Temporarily disabled due to issue (#499)")
```

**Review by jerrysky3 — APPROVED** [review]

**Pull request merged**